### ISSUE TITLE
Resolution issue

### ISSUE THREAD

**Issue by ptorrent** [Flam3rboy/puppeteer-stream#22]
Hello,

I'm using xvfb with this parameters:

-screen 0 1920x1080x16

And puppeteer-stream like that:

```
var screenWidth = 1920
var screenHeight = 1080
var xvfb 		= new Xvfb({
	silent: true,
    xvfb_args:["-screen", "0", screenWidth+"x"+screeHeight+"x16", "-ac"],
});

xvfb.startSync();
var browser  = await puppeteerStream.launch({
	defaultViewport: {
		width	: screenWidth ,
		height	: screenHeight 
	},
	args: ['--no-sandbox']
});
var stream = await puppeteerStream.getStream(page, { 
	audio: false, 
	video: true 
});
```

But I'm never able to get correct resolution.. If i print screen.width and screen.height (you can see it in the middle of screenshots), I correctyl have 1920x1080. With window.innerWidth / window.innerHeight I also have have good value ! Looks like an issue with the stream..

What I have (this is the video output):
![image](https://user-images.githubusercontent.com/6083644/139595858-0421fb81-22a1-43c9-b7a8-82e308758f34.png)

What I must have: (from browser):
![image](https://user-images.githubusercontent.com/6083644/139595887-dd1457e1-c808-4270-b182-3948689200f3.png)

One more issue:

I'm not able to restart the video, it's looks like there is no video duration, M I missing something ?

![image](https://user-images.githubusercontent.com/6083644/139596311-f75dc615-15ab-4a33-b93d-27ca64fe2c55.png)

Video cursor is always at 0. Don't know if it's an issue with windows video reader or.. Same issue with VLC:

![image](https://user-images.githubusercontent.com/6083644/139596351-57a6775b-ea0e-437b-9f41-6e22ec2ef2ae.png)

This is how I stop the record:

```
await stream.destroy();
file.close();
await browser.close(); 
xvfb.stopSync();
```
Don't know where to start to debug this issue, any idea?

Your help is welcome !

**Comment by ptorrent** [comment]
maybe related to https://github.com/Flam3rboy/puppeteer-stream/issues/15

**Comment by Flam3rboy** [comment]
resolved with #32
